src/bundler/bundle_v2.zig

```diff
@@ -2224,6 +2260,22 @@ pub const BundleV2 = struct {
                 //
                 // The file could be on disk.
                 if (strings.eqlComptime(resolve.import_record.namespace, "file")) {
+                    if (resolve.import_record.kind == .entry_point_build) {
+                        const resolved = this.transpiler.resolveEntryPoint(resolve.import_record.specifier) catch {
```

Comment:
This is not correct when using full-stack builds
```suggestion
                        const target = resolve.import_record.original_target;
                        const resolved = this.transpilerForTarget(target).resolveEntryPoint(resolve.import_record.specifier) catch {
```